### ISSUE TITLE
Внешняя обработка перестает компилироваться, если в ней использовать элементы стиля основной конфигурации

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#1357]
### Описание ошибки

Внешняя обработка перестает компилироваться, если в ней использовать элементы стиля основной конфигурации
Например, в основной конфигурации определен ЭлементСтиля1 (некоторый цвет). Выберем его в качестве цвета шрифта для поля ввода на форме внешней обработки, как показано на рис. ниже. И можно видеть, что обработка перестала компилироваться.

![image](https://github.com/1C-Company/1c-edt-issues/assets/54451084/dd52c5ce-88dd-414c-992b-86bb815f67ac)

### Как воспроизвести

1. Создать пустой проект
2. Создать в нем ЭлементСтиля1  (цвет)
3. Создать проект внешней обработки с выбранным основным проектом (создали в пункте 1)
4. Создать форму обработку
5. Создать реквизит формы обработки Реквизит1 с типом строка и перетащить его на форму (будет создано поле)
6. У поля ввода Реквизит1 для свойства ЦветТекста выбрать ЭлементСтиля1  основной конфигурации
7. Наблюдать пропажу обработки в каталоге bin

### Ожидаемое поведение

Либо обработка должна компилироваться, либо не позволять выбирать стили, которые ведут к такому поведению

### Лог рабочей области

не требуется

### Версия 1С:EDT

Ruby 2023.3.1

### Операционная система

Windows

**Comment by wolfandw** [comment]
На версии 2023.3.5 не воспроизводится. Проверьте настройки проекта внешней обработки в разделе "Настройки выгрузки" и для базового проекта должно быть создано приложение для разработки, связанное с ИБ.

**Comment by DmitryShvaika** [comment]
Да, на этой версии не воспроизводится - видимо исправили. Закрываю задачу. Если повторится - заведу новую.
